micrometer-core/src/main/java/io/micrometer/core/instrument/binder/grpc/DefaultGrpcServerObservationConvention.java

```diff
@@ -46,10 +46,17 @@ public KeyValues getLowCardinalityKeyValues(GrpcServerObservationContext context
         List<KeyValue> keyValues = new ArrayList<>();
         keyValues.add(LowCardinalityKeyNames.METHOD.withValue(context.getMethodName()));
         keyValues.add(LowCardinalityKeyNames.SERVICE.withValue(context.getServiceName()));
+        keyValues.add(LowCardinalityKeyNames.SYSTEM.withValue("grpc"));
         keyValues.add(LowCardinalityKeyNames.METHOD_TYPE.withValue(context.getMethodType().name()));
         if (context.getStatusCode() != null) {
             keyValues.add(LowCardinalityKeyNames.STATUS_CODE.withValue(context.getStatusCode().name()));
         }
+        if (context.getPeerName() != null) {
+            keyValues.add(LowCardinalityKeyNames.PEER_NAME.withValue(context.getPeerName()));
```

Comment:
I wonder if it would be better to use a value such as `UNKNOWN` if the peer name is null. What do you think?

Reply:
@shakuzen Yes, I agree giving `UNKNOWN` value is better rather than omitting the tag.
I have updated to specify `UNKNOWN` when value is not available(`null`).
Please check.